home.html

```diff
@@ -188,6 +188,23 @@ <h3 class="connect">
    </div>
     </div>
   </section>
+  <div class="contact_me_container"> 
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mpzbjpqb" method="POST" class="form" >
+        <div class="form_section">
+            <input type="text" name="name" placeholder="Name" class="form_input" maxlength="30" required>
+        </div>
+        <div>
+            <input type="email" name="email" placeholder="Email" class="form_input"  required>
+        </div>
+        <div>
+            <textarea class="form_input" placeholder="Write your message here" maxlength="500" required></textarea>
+        </div>
+        <div>
+            <input type="submit" class="project1" value="Get it Touch"/>
+        </div>
```

Comment:
- [ ] [Optional] Kindly implement button interactions to the contact-form button
![Capture](https://user-images.githubusercontent.com/85095702/160753020-02303e61-a865-4175-8532-3b830eb8da77.PNG)